src/gf_getfld.f

```diff
@@ -1,229 +1,196 @@
+!>    @file
+!>    @brief Contains subroutines returns the Grid Definition,
+!>    and Product Definition for a given data field.
+!>    @author Stephen Gilbert @date 2000-05-26
+!>
+
+!>    This subroutine returns the Grid Definition, Product Definition,
+!>    Bit-map ( if applicable ), and the unpacked data for a given data
```

Comment:
Remove extra space inside the parens, so:

 `Bit-map (if applicable), and the unpacked data for a given data
`

Reply:
Thanks